gcp/shared/utils/config_models.py

```
@@ -164,6 +166,49 @@ class FollowupSurveysConfig(BaseModel):
+        for key, times in v.items():
+            if len(times) != 3:
+                raise ValueError(
+                    f"shift '{key}' must have exactly 3 times, got {len(times)}"
+                )
+            for t in times:
+                try:
+                    time.fromisoformat(t)
+                except ValueError:
+                    raise ValueError(
+                        f"shift '{key}' time '{t}' is not a valid HH:MM time"
+                    )
```

Ruff (0.15.15)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)